test/views/ViewSpecBase.scala

```diff
@@ -93,6 +109,31 @@ trait ViewSpecBase extends SpecBase {
   def assertNotRenderedByCssSelector(doc: Document, cssSelector: String): Assertion =
     assert(doc.select(cssSelector).isEmpty, "\n\nElement " + cssSelector + " was rendered on the page.\n")
 
+  def assertRenderedByTagWithAttributes(doc: Document, tagName: String, attributes: (String, String)*): Assertion = {
```

Comment:
This method and some others are unused. Is there a reason why they are still in here? For future use or?

Reply:
As discussed, most were copied across were potentially useful and will delete unused as views now have coverage